src/amber/server/env.cr

```diff
@@ -1,33 +1,38 @@
 module Amber
   def self.env
-    Env.new(Settings.env.not_nil!.downcase)
+    @@env ||= Env.new(ENV["AMBER_ENV"]? || "development") # Would be best to only define this once.
   end
 
   class Env
-    def initialize(@env : String | Symbol)
+    alias EnvType = String | Symbol 
+
+    def initialize(@env : String) end
+
+    def is?(environment : EnvType)
```

Comment:
I think we can remove this method with the new addition of the `==`

Reply:
Again I'm fine with removing. You guys had wanted it yesterday though so I didn't want to just go delete it. Also I sort of though of it as [] and fetch in hash. https://github.com/crystal-lang/crystal/blob/d1f8c42f82d3e6491cb7b3b536984c19a51b94ba/src/hash.cr#L59

Reply:
WDYT?

Reply:
@elorest you made a good improvement on this. Yesterday it was `Amber::Env.is?`, but now it is `Amber.env ==` which is much more natural 🎉

Besides, personally I'm indifferent to `[]` approach.

Reply:
I think we can just remove it. We covered all the current use cases. Less is more in this case.